eslint warnings coming from Gatsby

```
warn ESLintError: 
/gatsby-theme-carbon-1.19.0/packages/gatsby-theme-carbon/src/components/GlobalSearch/Menu.js
 48:9 warning Non-interactive elements should not be assigned interactive
roles jsx-a11y/no-noninteractive-element-to-interactive-role
warn ESLintError: 
/gatsby-theme-carbon-1.19.0/packages/gatsby-theme-carbon/src/components/Tabs/Tabs.js
 45:36 warning Non-interactive elements should not be assigned interactive
roles jsx-a11y/no-noninteractive-element-to-interactive-role
```

displayed in console on fresh dev from master

--- a/packages/gatsby-theme-carbon/src/components/GlobalSearch/Menu.js
+++ b/packages/gatsby-theme-carbon/src/components/GlobalSearch/Menu.js
@@ -1,3 +1,7 @@
+// Gatsby doesn't include the recommended exceptions to this rule
+/* eslint-disable jsx-a11y/no-noninteractive-element-to-interactive-role */
+// https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-to-interactive-role.md#rule-details
+
 import React, { useEffect, createContext, useContext } from 'react';
 import { Link } from 'gatsby';
 import cx from 'classnames';
@@ -42,26 +46,24 @@ const Menu = ({ results, onKeyDown }) => {
   });
 
   return (
-    <>
-      <ul
-        aria-labelledby="search-label"
-        role="menu"
-        id="search-menu"
-        className={cx(list, {
-          [hidden]: results.length === 0,
-        })}
-      >
-        {resultsWithTabAdded.map((page, index) => (
-          <MenuItem
-            id={`menu-item-${index}`}
-            onKeyDown={onKeyDown}
-            key={page.path}
-            index={index}
-            page={page}
-          />
-        ))}
-      </ul>
-    </>
+    <ul
+      aria-labelledby="search-label"
+      role="menu"
+      id="search-menu"
+      className={cx(list, {
+        [hidden]: results.length === 0,
+      })}
+    >
+      {resultsWithTabAdded.map((page, index) => (
+        <MenuItem
+          id={`menu-item-${index}`}
+          onKeyDown={onKeyDown}
+          key={page.path}
+          index={index}
+          page={page}
+        />
+      ))}
+    </ul>
   );
 };
 

--- a/packages/gatsby-theme-carbon/src/components/Tabs/Tabs.js
+++ b/packages/gatsby-theme-carbon/src/components/Tabs/Tabs.js
@@ -1,3 +1,7 @@
+// Gatsby doesn't include the recommended exceptions to this rule
+/* eslint-disable jsx-a11y/no-noninteractive-element-to-interactive-role */
+// https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-to-interactive-role.md#rule-details
+
 import React, {
   useContext,
   createContext,

